util/n64/rominfo.py

```diff
@@ -42,52 +47,61 @@
     0x59: "European",
 }
 
+
+class CIC:
```

Comment:
I like this change, but we might as well use a dataclass for it (@dataclass) 

```python

@dataclass
class CIC:
    ntsc_name: str
    pal_name: str
    offset: int
```

Reply:
Done, I do boomer python in some aspects 🧓